cli/analyse.py

```diff
@@ -102,35 +111,101 @@ def observable(
         """,
     )
     # first step: ask analysis availability
+    path = pathlib.Path(runtime_config)
+    if path.exists():
+        runtime_config = json_load(open(runtime_config))
+    else:
+        runtime_config = {}
     ans = ctx.obj.send_observable_analysis_request(
         analyzers_requested=analyzers_list,
         observable_name=value,
         force_privacy=force_privacy,
         private_job=private_job,
         disable_external_analyzers=disable_external_analyzers,
         run_all_available_analyzers=run_all,
+        runtime_configuration=runtime_config,
     )
     warnings = ans["warnings"]
     ctx.obj.logger.info(
         f"""New Job running..
         ID: {ans['job_id']} | Status: [underline pink]{ans['status']}[/].
-        Got {len(warnings)} warnings: [italic red]{warnings if warnings else None}[/]
+        Received {len(warnings)} warnings: [italic red]
+        {warnings if warnings else None}[/]
     """
     )
 
 
 @analyse.command(short_help="Send analysis request for a file")
 @click.argument("filename", type=click.Path(exists=True))
 @add_options(__analyse_options)
+@click.option(
+    "-b",
+    "--binary",
+    type=bool,
+    default=True,
+    help="True if File is Binary False otherwise",
+)
 @click.pass_context
 def file(
     ctx: ClickContext,
     filename,
+    binary,
     analyzers_list,
     run_all,
     force_privacy,
     private_job,
     disable_external_analyzers,
     check,
+    runtime_config,
 ):
-    pass
+    if analyzers_list and run_all:
+        ctx.obj.logger.warn(
+            """
+            Can't use -al and -aa options together. See usage with -h.
+            """
+        )
+        ctx.exit(-1)
+    if not (analyzers_list or run_all):
+        ctx.obj.logger.warn(
+            """
+            Either one of -al, -aa must be specified. See usage with -h.
+            """,
+        )
+        ctx.exit(-1)
+    analyzers = analyzers_list if analyzers_list else "all available analyzers"
+    ctx.obj.logger.info(
+        f"""Requesting analysis..
+        file: [bold blue underline]{filename}[/]
+        analyzers: [italic green]{analyzers}[/]
+        """,
+    )
+    # first step: ask analysis availability
+    read_mode = "rb" if binary else "r"
```

Comment:
not needed, we would always read any given file in binary mode since it will be transmitted over HTTP